src/SqlStreamStore.Postgres/PgSqlScripts/AppendToStream.sql

```diff
@@ -105,9 +105,10 @@ BEGIN
                                      created_utc,
                                      type,
                                      json_data,
-                                     json_metadata)
+                                     json_metadata,
+                                     transaction_id)
     SELECT m.message_id, _stream_id_internal, _current_version + (row_number()
-        over ()) :: int, _created_utc, m.type, m.json_data, m.json_metadata
+        over ()) :: int, _created_utc, m.type, m.json_data, m.json_metadata, pg_current_xact_id()
```

Comment:
So explained IRL.
'AppendToStream' is not a read-only operation. We are rather sure this has the intention to write something in the DB.

So I'm fine the transaction id gets assigned
The other one will return NULL since we didn't do a write action yet.

And I need the transactionId to be in the 'first' insert.